kustomize/private/tools/kustomize/toolchain.bzl

```diff
@@ -3,6 +3,14 @@
 visibility("public")
 
 _TOOLS_BY_RELEASE = {
+    "v5.1.0": {
+        struct(os = "darwin", arch = "amd64"): "08664a17820138a68b7cbe302b1b63f4ec19c6e0838385f789ee0470f026ba25",
+        # NB: There is no Darwin build available for the ARM64 architecture.
```

Comment:
I'm going to hold on committing this patch pending [discussion in the "kustomize" channel](https://kubernetes.slack.com/archives/C9A5ALABG/p1687266804584799) of the "Kubernetes" Slack team.

Reply:
Watch kubernetes-sigs/kustomize#5220 for progress.